modules/mobile-agents/android/client/src/org/wso2/mdm/agent/api/Root.java

```diff
@@ -55,11 +55,12 @@ public boolean isDeviceRooted() {
 	 * @return - Device root status by build tags.
 	 */
 	public boolean checkRootByBuildTags() {
-        	String buildTags = android.os.Build.TAGS;
+
+		String buildTags = android.os.Build.TAGS;
 		if (buildTags != null && buildTags.contains(SU_TAG)) {
-            		if (Constants.DEBUG_MODE_ENABLED){
-                		Log.d(TAG, "Build tags found");
-            		}
+			if (Constants.DEBUG_MODE_ENABLED){
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/b0c66506d24fb5288300bec4224f1eaa58c7b426]